cmd/kubectl-gadget/deploy.go

```diff
@@ -128,6 +128,10 @@ rules:
   resources: ["traces", "traces/status"]
   # For traces, we need all rights on them as we define this resource.
   verbs: ["delete", "deletecollection", "get", "list", "patch", "create", "update", "watch"]
+- apiGroups: ["*"]
```

Comment:
Why is it not `""`? Is there any difference?

Reply:
`""` indicates the core API group, for instance `core/v1`, and namespaces, nodes, pods and services use it, check [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#-strong-api-groups-strong-). Take into account that the core group is not specified as part of the apiVersion field and that's why we just see `apiVersion: v1` on the resource description.

On the other hand, the resources we are interested in for the seccomp use different API groups. In particular, Job and CronJob use `batch/v1`, DaemonSet, Deployment, ReplicaSet and StatefulSet use `app/v1`, while ReplicationController is the only one to use `core/v1`. 

For the sake of simplicity, I decided to use a wildcard `"*"` and avoid adding three independent rules separated by API group. What do you think?

Reply:
Alright, makes sense.